sharc/sharc_definitions.py

```diff
@@ -8,6 +8,7 @@
     # "FSS_ES",
     # "FS",
     # "RAS",
+    "MSS_SS",
     "SINGLE_EARTH_STATION",
-    "SINGLE_SPACE_STATION",
+    "MSS_D2D"
```

Comment:
```suggestion
    "MSS_D2D",
    "SINGLE_SPACE_STATION",
```
Estava apagando sem querer